packit_service/worker/handlers/fedmsg_handlers.py

```diff
@@ -142,20 +136,64 @@ def run(self) -> HandlerResults:
             # rev is a commit
             # we use branch on purpose so we get the latest thing
             # TODO: check if rev is HEAD on {branch}, warn then?
-            dist_git_branch=self.distgit_event.branch,
+            dist_git_branch=self.branch,
             upstream_branch="master",  # TODO: this should be configurable
         )
         return HandlerResults(success=True, details={})
 
 
+class CoprBuildHandler(FedmsgHandler):
```

Comment:
Maybe add something about results to the name? (So we easily differentiate the ones that trigger build and ones that update the status.)

Reply:
what do you think about `CoprBuildStateHandler`?

Reply:
Can be. OR `CoprBuildReportHandler` to match the Koji one? Or `AbstractCoprBuildReportHandler` to also show that it needs to be inherited.